src/main/java/algorithms/curated170/medium/NumberOfDistinctIslands.java

```diff
@@ -0,0 +1,88 @@
+package algorithms.curated170.medium;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+public class NumberOfDistinctIslands {
+    
+    Set<String> islands;
+    int[][] grid;
+    int xDim;
+    int yDim;
+    public int numOfIslands(int[][] grid)
```

Comment:
Can we alter the method name as numDistinctIslands to be more compatible with LeetCode?